.github/workflows/validation.yml

```diff
@@ -0,0 +1,37 @@
+name: Validation
+
+on: pull_request
```

Comment:
Can you synchronize with `graphql-js` config: https://github.com/graphql/graphql-js/blob/master/.github/workflows/ci.yml#L2
For example, we also need to run this flow on `push`.
Also having the same names and keys helps with switching between projects.
At the same time, it's totally ok to have stuff that is unique to this project in places where it makes sense.

Reply:
yea, push is perfectly fine. this is depending on if you want to use the same workflow for `master/main` and a `pr`  so I was conservative